package-list.json

```diff
@@ -13,14 +13,23 @@
       "fhirversion": "4.0.1",
       "current": true
     },
```

Comment:
Overall, is this file actually maintained in GitHub? I can't tell since the 0.3.0-ballot has already been published without this file being changed. Where is the source of truth?